web.esphome.io/src/dashboard/device-card.ts

```diff
@@ -53,6 +47,7 @@ class EWDeviceCard extends LitElement {
             <mwc-list-item>Configure Wi-Fi</mwc-list-item>
             <mwc-list-item>Disconnect</mwc-list-item>
           </esphome-button-menu>
+        </div>
```

Comment:
Please split this out into a new PR.

Reply:
Done in https://github.com/esphome/dashboard/pull/309